lib/mimicry/open_api/example.ex

```diff
@@ -9,13 +9,16 @@ defmodule Mimicry.OpenAPI.Example do
   chooses an example based on the defined examples for the response and the spec (for references)
   """
   @spec choose(Response.t(), String.t() | :random, Specification.t()) ::
-          {:ok, map()} | {:error, :not_found}
+          {:ok, map()} | {:error, :not_found} | {:error, :no_examples}
```

Comment:
I haven't touched Elixir for a while now, esp not the specs.
So a quick question: Wouldn't it be possible to specify the errors more concisely like this?
```suggestion
          {:ok, map()} | {:error, :not_found | :no_examples}
```
Or is it possible but bad style?

Reply:
I think it is better actually, but not completely sure. The next steps for errors would actually be to introduce specific Error-structs to match on, instead of relying on hard coded atoms.